pkg/audit/manager.go

```diff
@@ -244,7 +244,11 @@ func (am *Manager) auditResources(
 
 	serverResourceLists, err := discoveryClient.ServerPreferredResources()
 	if err != nil {
-		return err
+		if discovery.IsGroupDiscoveryFailedError(err) {
```

Comment:
Can we have a generic catch error case to protect from similar kind of errors in future?

Reply:
What other scenarios are you thinking off? If we can't get resources here, it'll bubble up the error.

Reply:
This particular scenario sounds good. Other scenario to check for is OPA review errors. On any OPA review error, currently I see we are not writing audit results. Please check this scenario and make a fix if needed.